app/views/proposal/show.html.haml

```diff
@@ -47,6 +47,15 @@
       .row
         .col-md-12
           %p
+            - if @event.state == 'confirmed' && !@event.intersecting_events.withdrawn.empty?
```

Comment:
- This could be moved to the [top of the page](https://github.com/nishanthvijayan/osem/blob/5585eafbcd6e6db3679552c54101e1ea3bf17dec/app/views/proposal/show.html.haml#L37)
 - Like Ana suggested, the coloured label could be displayed here as well (before the title)